vpr/src/power/power.cpp

```diff
@@ -982,9 +982,9 @@ static void power_usage_routing(t_power_usage* power_usage,
                 connectionbox_fanout = 0;
                 switchbox_fanout = 0;
                 for (t_edge_size iedge = 0; iedge < node.num_edges(); iedge++) {
-                    if (node.edge_switch(iedge) == routing_arch->wire_to_rr_ipin_switch) {
+                    if (rr_graph.edge_switch(RRNodeId(rr_node_idx), iedge) == routing_arch->wire_to_rr_ipin_switch) {
```

Comment:
In LINE 861, the ``RRNodeId(rr_node_idx)`` is already defined as another variable ``rr_node``.
I believe you can safely replace it here and in the rest of this function.